src/bots/jonas.ts

```diff
@@ -5,13 +5,15 @@
 // - je meer geld hebt dan het laatste hoogste bod
 // - je niet het hoogste bod hebt
 // komt eigenlijk gewoon neer als het nog zin heeft om een bod te doen dus ja
+import type {PlayerKey} from "@/models/auction.models.ts";
```

Comment:
Ja dit leek me onhandig voor nu omdat het dan niet met alleen dit bestand zichtbaar is wat je allemaal door kan krijgen snap je